spec/components/candidate_interface/new_reference_status_line_component_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::NewReferenceStatusLineComponent, type: :component do
+  it 'renders a status line for request_bounced events', with_audited: true do
+    reference = create(:reference, :not_requested_yet, email_address: 'example@email.com')
+    reference.email_bounced!
+
+    result = render_inline(described_class.new(reference))
+
+    list_item = result.css('p').first
+    expect(list_item.text).to include 'Email could not be sent - check email address and send again'
+  end
+
+  it 'renders a status line for request_sent events', with_audited: true do
+    reference = create(:reference, :not_requested_yet, email_address: 'example@email.com')
+    reference.feedback_requested!
+
+    result = render_inline(described_class.new(reference))
+
+    list_item = result.css('p').first
+    expect(list_item.text).to include "Request sent on #{Time.zone.now.to_fs(:govuk_date)}"
+  end
+
+  it 'renders cancel request link for reference feedback_status that is feedback_requested', with_audited: true do
+    reference = create(:reference, :not_requested_yet)
+    reference.feedback_requested!
+
+    render_inline(described_class.new(reference))
+
+    expect(rendered_component).to have_text 'or cancel request'
```

Comment:
Might make sense to also check for the reminder link here too?

```suggestion
    expect(rendered_component).to have_text 'send a reminder or cancel request'
```